seqr/views/utils/permissions_utils.py

```diff
@@ -150,21 +152,37 @@ def _get_analyst_projects():
     return ProjectCategory.objects.get(name=ANALYST_PROJECT_CATEGORY).projects.all()
 
 
-def get_projects_user_can_view(user):
-    if user_is_data_manager(user):
-        return Project.objects.all()
+def get_project_guids_user_can_view(user):
+    cache_key = 'projects__{}'.format(user)
+    project_guids = safe_redis_get_json(cache_key)
+    if project_guids is not None:
+        return project_guids
```

Comment:
How about caching the project guids permitted by AnVIL only. It seems doesn't make sense for the location permissions won't take effect instantly.

Reply:
I disagree. The performance boost for caching all guids is more than worth a small lag time in local project permissions, especially since we are actively working to get rid of local project permissions

Reply:
Okay